scripts/lib.star

```diff
@@ -286,6 +315,11 @@ def init_steps(edition, platform, is_downstream=False, install_deps=True):
         },
     ]
 
+    if version_mode == 'master':
+        steps.append(slack_step(channel='grafana-ci-failures'))
```

Comment:
I've already setup a webhook secret for the #grafana-ci slack channel. If you rather would like to use this one let me know and I'll delete my channel and webhook token

Reply:
@marefr Aha... I saw that secret and got confused where it came from, so I replaced it with my own. What channel had you set up already?

Reply:
Merging now, since we can discuss which webhook to use separately :)